jetstream/tas_api.py

```diff
@@ -8,6 +8,7 @@
 
 
 def tacc_api_post(url, post_data, username=None, password=None):
+    raise NotImplementedError  # TODO: DON'T COMMIT
```

Comment:
Don't worry about this. :) This is just while I test so I know that I am not accidentally touching the API. In a near future commit I'll actually make a mock for this. Same below in `tacc_api_get`